### ISSUE TITLE
Messaging Diagram(s)

### ISSUE THREAD

**Issue by ptheywood** [FLAMEGPU/FLAMEGPU2_userguide#46]
> *TODO: Diagram demonstrating messaging structure*

From `src/guild/3-behaviour-definition/4-agent-communication.rst`

**Comment by Robadob** [comment]
I don't believe this is present in the docs anymore since the rework, but there's still probably an argument for adding something about message diagrams/philosphy somewhere.
